styles/Home.module.css

```diff
@@ -1,10 +1,11 @@
 .container {
   min-height: 100vh;
-  padding: 0 0.5rem;
+  /* padding: 0 0.5rem; 
   display: flex;
   flex-direction: column;
   justify-content: center;
-  align-items: center;
+  align-items: center; */
```

Comment:
It's a good practice to remove code that we're not using.

Reply:
Thank you! I made the change